.changeset/early-forks-change.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sketch-hq/sketch-file-format': minor
+---
+
+Added didManuallyDisableResizingContent property to stop artboards from
+automatically enabling resizesContent.
```

Comment:
From a file format consumer angle it might be better to rephrase it a little explaining the relationship with `resizesContent`, not the behaviour. The file doesn't have a behaviour as such. Perhaps something like:

```suggestion
Added `didManuallyDisableResizingContent` which, if set to `true`, overrides and disables `resizesContent`.
```

Reply:
That's not quite what it does though - it doesn't override and disable `resizesContent`. It does stop `resizesContent` from being set to `true` as a side effect of enabling a constraint on a child layer in the inspector.

Reply:
```suggestion
Added `didManuallyDisableResizingContent` property to stop `resizesContent` from being enabled automatically when setting a constraint on a child layer.
```

Reply:
How about this?